scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -169,11 +172,12 @@ public boolean doCheckAndSend(boolean withAttachment)
                 } else if ((jobStatusList.contains(eventFinishedWithErrorsName) ||
                             jobStatusList.contains(eventFinishedWithErrorsMethod))) {
                     // check if any tasks have issues
-                    if (hasTasksWithIssues()) {
-                        sendEmail(withAttachment, true, false);
+                    if (jobEmailStatus.hasTasksWithErrors()) {
+                        jobEmailStatus.setWithErrors(true);
```

Comment:
The conditional statement is not required anymore

Reply:
Why systematically have setWithErrors() set to true when hasTasksWithErrors() is valued w/ true outside of the class ? This logic should be implemented inside the JobEmailStatus class

Reply:
this is a bit strange indeed, isn't jobEmailStatus.hasTasksWithErrors() sufficient ?

Reply:
done